src/components/backend-ai-resource-policy-list.ts

```diff
@@ -209,88 +243,92 @@ export default class BackendAIResourcePolicyList extends BackendAIPage {
-                  ${this.gpu_metric.map(item => html`
-                    <paper-item value="${item}">${item}</paper-item>
-                  `)}
-                  </paper-listbox>
-                </paper-dropdown-menu>
-                <paper-dropdown-menu id="fgpu-resource" label="fgpu">
-                  <paper-listbox slot="dropdown-content" selected="0">
-                  ${this.fgpu_metric.map(item => html`
-                    <paper-item value="${item}">${item}</paper-item>
-                  `)}
-                  </paper-listbox>
-                </paper-dropdown-menu>
+                  <div class="vertical layout" style="width:75px; margin: 0px 10px 0px 0px;">
+                    <wl-label>CPU</wl-label>
+                    <wl-textfield id="cpu-resource" type="number"
+                    @change="${(e) =>this._validateResourceInput(e)}"
+                    ></wl-textfield>
+                      <wl-label class="unlimited">
+                        <wl-checkbox @change="${(e) =>this._toggleCheckbox(e)}" style="border-width: 1px;"
+                        ></wl-checkbox>
+                        Unlimited
+                      </wl-label>
+                  </div>
+                  <div class="vertical layout" style="width:75px; margin: 0px 10px 0px 10px;">
+                    <wl-label>RAM(GB)</wl-label>
+                    <wl-textfield id="ram-resource" type="number"
+                    @change="${(e) =>this._validateResourceInput(e)}"></wl-textfield>
+                    <wl-label class="unlimited">
+                      <wl-checkbox @change="${(e) =>this._toggleCheckbox(e)}" style="border-width: 1px;"></wl-checkbox>
+                      Unlimited
+                    </wl-label>
+                  </div>
+                  <div class="vertical layout" style="width:75px; margin: 0px 10px 0px 10px;">
+                    <wl-label>GPU</wl-label>
+                    <wl-textfield id="gpu-resource" type="number" @change="${(e) =>this._validateResourceInput(e)}"></wl-textfield>
+                    <wl-label class="unlimited">
+                      <wl-checkbox @change="${(e) =>this._toggleCheckbox(e)}" style="border-width: 1px;"></wl-checkbox>
+                      Unlimited
+                    </wl-label>
+                  </div>
+                  <div class="vertical layout" style="width:75px; margin: 0px 0px 0px 10px;">
+                    <wl-label>fGPU</wl-label>
+                    <wl-textfield id="fgpu-resource" type="number" @change="${(e) =>this._validateResourceInput(e)}"></wl-textfield>
+                    <wl-label class="unlimited">
+                      <wl-checkbox @change="${(e) =>this._toggleCheckbox(e)}" style="border-width: 1px;"></wl-checkbox>
+                      Unlimited
+                    </wl-label>
+                  </div>
               </div>
-              <h4>Sessions</h4>
+                <h4>Sessions</h4>
```

Comment:
Why spaces are added? I think `h4` tag is in parallel with `div` before and after it.